build-sensor/include/step4.h

```diff
@@ -0,0 +1,75 @@
+
+WiFiClient espClient;
+PubSubClient client(espClient);
```

Comment:
This bothers me. This requires:
#include <PubSubClient.h>
Which I had to include once globally in Main.cpp
Why cant I include it in both step4.h and step5.h and define it as needed for just these classes.
I was also unable to contain its scope within the class.
@wilvk Any ideas ?

Reply:
Further, these 2 definitions are then later used in step5.h as I cannot duplicate the definitions in the global namespace.

Reply:
hey . @tfindlay-au , sorry, I missed this - will check it out tonight.